pdns/slavecommunicator.cc

```diff
@@ -650,6 +650,7 @@ struct SlaveSenderReceiver
     uint32_t theirSerial;
     uint32_t theirInception;
     uint32_t theirExpire;
+    string master;
```

Comment:
This should probably be a ComboAddress, if that integrates somewhat well with the rest (I know SlaveCommunicator likes casting between string and CA a lot...)

Reply:
The "master" variable is used to hold data from DomainInfo.masters which is:

```
struct DomainInfo
{
...
  vector<string> masters;
```

So I just used the same data type.